articles/logic-apps/error-exception-handling.md

```diff
@@ -195,67 +195,34 @@ For example, to run the Office 365 Outlook **Send an email** action after the Ex
 
 ### Change "run after" behavior in the designer
 
-### [Standard](#tab/standard)
-
 1. In the [Azure portal](https://portal.azure.com), open the logic app workflow in the designer.
 
-1. On the designer, select the action shape. On the details pane, select **Run After**.
+1. On the designer, select the action shape. On the details pane, select **Settings**.
 
-   ![Screenshot showing Standard workflow designer and current action details pane with "Run After" selected.](./media/error-exception-handling/configure-run-after-standard.png)
+1. The **Run After** section in the **Settings** pane shows the predecessor action for the currently selected action.
 
-   The **Run After** pane shows the predecessor action for the currently selected action.
+   ![Screenshot showing the designer and current action details pane with "Settings" selected.](./media/error-exception-handling/configure-run-after.png)
 
-   ![Screenshot showing Standard designer, current action, and "run after" status for predecessor action.](./media/error-exception-handling/predecessor-action-standard.png)
 
 1. Expand the predecessor action node to view all the "run after" statuses.
 
    By default, the "run after" status is set to **is successful**. So, the predecessor action must run successfully before the currently selected action can run.
 
-   ![Screenshot showing Standard designer, current action, and default "run after" set to "is successful".](./media/error-exception-handling/change-run-after-status-standard.png)
+   ![Screenshot showing the designer, current action, and default "run after" set to "is successful".](./media/error-exception-handling/change-run-after-status.png)
 
 1. Change the "run after" behavior to the status that you want. Make sure that you first select an option before you clear the default option. You have to always have at least one option selected.
 
    The following example selects **has failed**.
 
-   ![Screenshot showing Standard designer, current action, and "run after" set to "has failed".](./media/error-exception-handling/failed-run-after-status-standard.png)
+   ![Screenshot showing the designer, current action, and "run after" set to "has failed".](./media/error-exception-handling/failed-run-after-status.png)
```

Comment:
Alt text is limited, so we can omit some words. We can no longer use special characters, such as quotation marks. This syntax automatically adds the required border around screenshots.

```suggestion
   :::image type="content" source="media/error-exception-handling/failed-run-after-status.png" alt-text="Screenshot shows current action with run after behavior set to Has failed.":::
```